android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPAliases.kt

```diff
@@ -23,7 +23,6 @@ typealias CPListItem = Row
 typealias CPListTemplate = ListTemplate
```

Comment:
is this file needed if the package is only supporting CarPlay and not Android Auto?

Reply:
Need to remove map related things because of the here_sdk imports. Did not get it running with the Map templates.